DEPLOYMENT_TYPES.md

```diff
@@ -14,9 +17,12 @@ The table below outlines the defaults per `deployment_type`.
 | **Image Streams**                                               | centos                                   | rhel + xpaas                           | rhel                             |
 
 
-**NOTE** `enterprise` deployment type is used for OpenShift Enterprise version
-3.0.x OpenShift Enterprise deployments utilizing version 3.1 and later will
-make use of the new `openshift-enterprise` deployment type.  Additional work to
-migrate between the two will be forthcoming.
+**NOTES**:
+
+- `enterprise` deployment type is used for OpenShift Enterprise version
+3.0.x.
+- `openshift-enterprise` deployment type is used for OpenShift Enterprise (and now OpenShift Container Platform) version 3.1 and later.
 
 
+# REVIEW: notes, what versions use what
+# REVIEW: table, what is the registry_url for openshift-enterprise -- isn't it openshift3/ose-...?
```

Comment:
Yeah, you got it right. note-to-self --  check outdated comments as part of review.